packages/wallet/src/W3mFrameProvider.ts

```diff
@@ -422,7 +422,7 @@ export class W3mFrameProvider {
     })
   }
 
-  public async switchNetwork(chainId: number) {
+  public async switchNetwork(chainId: number | string) {
```

Comment:
Could we have `ChainId` type somewhere in common place? I guess we already have that type but should be for something else. Let's double check if we could use in these cases, to not repeate type definitions